packages/apps/plugins/plugin-presenter/src/PresenterPlugin.tsx

```diff
@@ -22,6 +23,11 @@ type PresenterState = {
   presenting: boolean;
 };
 
+const isLocalMarkdownContent = (data: unknown): data is { content: EditorModel } =>
```

Comment:
@wittjosiah this is most certainly incorrect. could probably be more terse. move back into `util.tsx`? although having it in `util.tsx` is probably what broke this in the first place - wasn't clear all of the places that it's being used.